html/pfappserver/lib/pfappserver/Form/Config/ProfileCommon.pm

```diff
@@ -428,6 +428,22 @@ has_field 'access_registration_when_registered' =>
              help => 'This allows already registered users to be able to re-register their device by first accessing the status page and then accessing the portal. This is useful to allow users to extend their access even though they are already registered.' },
   );
 
+=head2 status
+
+The status of the profile if it is enabled or disabled
+
+=cut
+
+has_field 'status' =>
+  (
+   type => 'Toggle',
+   label => 'Profile is enable/disabled',
+   checkbox_value => 'enabled',
+   unchecked_value => 'disabled',
+   tags => { after_element => \&help,
+             help => 'If profile is disabled it will not be presented to the user' },
```

Comment:
Not really true, a connection profile is instantiate in multiples cases.
So something like that: If profile is disabled it will not be used.